src/sdgio-edit.owl

```diff
@@ -158,6 +164,63 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
 Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020236>))
 Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020237>))
 Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020300>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020308>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020309>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020310>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020311>))
```

Comment:
removing this process as it is very specific. 
Instances of this class will be mapped to the 'development of natural resource acquisition capacity' class and to agricultural system